src/Appwrite/Extend/Exception.php

```diff
@@ -164,6 +171,18 @@ class Exception extends \Exception
     /** Webhooks */
     public const WEBHOOK_NOT_FOUND                 = 'webhook_not_found';
 
+    /** Router */
+    public const ROUTER_UNKNOWN_HOST                = 'router_unknown_host';
+    public const ROUTER_INVALID_TYPE                = 'router_unknown_type';
+
+    /** Proxy */
+    public const RULE_CONFIGURATION_MISSING         = 'rule_configuration_missing';
```

Comment:
This kind of metadata should always be there and we should ensure it with regular framework validators. If its still missing for some reason, that can just be a server error that we report back to our error logging solution.

Reply:
Removed exception, used server error insead.